untrusted/dcap/src/lib.rs

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+//! Rust wrappers for DCAP (Data Center Attestation Primitives) quote
+//! generation
+
+use mc_sgx_dcap_sys::{
+    quote3_error_t, sgx_qe_cleanup_by_policy, sgx_qe_get_quote, sgx_qe_get_quote_size,
+    sgx_qe_get_target_info, sgx_ql_path_type_t, sgx_ql_set_path, sgx_report_t, sgx_target_info_t,
+};
+use mc_sgx_urts::Enclave;
+use std::ffi::CString;
+use std::mem;
+use std::mem::MaybeUninit;
+
+/// Errors generating quotes
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    // An error provided from the SGX SDK
+    SgxStatus(quote3_error_t),
```

Comment:
Looks like there was an issue in bindgen, https://github.com/rust-lang/rust-bindgen/pull/2144, which was released yesterday... 
Brought the update to bindgen in to get hex output for now.
Created #13, I think it will most likely take some libclang parsing of the enums ourselves to generate the lookup table so leaving it for a follow on issue.